modules/hivemq-edge-module-modbus/src/main/java/com/hivemq/edge/adapters/modbus/ModbusProtocolAdapter.java

```diff
@@ -56,9 +53,10 @@ public class ModbusProtocolAdapter implements BatchPollingProtocolAdapter {
     private final @NotNull ProtocolAdapterState protocolAdapterState;
 
     private final @NotNull ModbusClient modbusClient;
-    private final @NotNull Map<String, Object> lastSamples = new ConcurrentHashMap<>();
+    private final @NotNull PublishChangedDataOnlyHandler lastSamples = new PublishChangedDataOnlyHandler();
```

Comment:
It's better to rename the variable to match the type.

Reply:
That was sloppy from me.
I changed the name :)